examples/transit_vnet_dedicated/example.tfvars

```diff
@@ -0,0 +1,86 @@
+location             = "East US 2"
+resource_group_name  = "example-rg"
+virtual_network_name = "vnet-vmseries"
+address_space        = ["10.110.0.0/16"]
+
+network_security_groups = {
+  "sg-mgmt"    = {}
+  "sg-private" = {}
+  "sg-public"  = {}
+}
+
+allow_inbound_mgmt_ips = [
+  "191.191.191.191", # Put your own public IP address here
+  "10.255.0.0/24",   # Example Panorama access
+]
+
+route_tables = {
+  "udr-private" = {
+    routes = {
+      default = {
+        address_prefix         = "0.0.0.0/0"
+        next_hop_type          = "VirtualAppliance"
+        next_hop_in_ip_address = "10.110.0.21"
+      }
+    }
+  }
+}
+
+subnets = {
+  "subnet-mgmt" = {
+    address_prefixes       = ["10.110.255.0/24"]
+    network_security_group = "sg-mgmt"
+  }
+  "subnet-private" = {
+    address_prefixes       = ["10.110.0.0/24"]
+    network_security_group = "sg-private"
+    route_table            = "udr-private"
+  }
+  "subnet-public" = {
+    address_prefixes       = ["10.110.129.0/24"]
+    network_security_group = "sg-public"
+  }
+}
+
+frontend_ips = {
+  "frontend01" = {
+    create_public_ip = true
+    rules = {
+      "balancessh" = {
+        protocol = "Tcp"
+        port     = 22
+      }
+    }
+  }
+}
+
+inbound_vmseries = {
+  "inboundfw00" = { avzone = 1 }
+  "inboundfw01" = { avzone = 2 }
+}
+
+inbound_vmseries_version = "10.0.4"
```

Comment:
Cannot we assume that all vmseries (inbound and outbound) have the same version and vm_size since they must have the same sku?

Reply:
We can assume it, sure.